operator/api/server.go

```diff
@@ -153,7 +153,7 @@ func (s *server) Start(ctx cell.HookContext) error {
 
```

Comment:
@aanm I see that all `label:feature/slog` PRs are now merged, do you still have planned work around the `slog` migration or can I start looking into rebasing this PR?

Reply:
sorry, this will be the last one https://github.com/cilium/cilium/pull/39889